extensions/amp-ad/amp-ad.md

```diff
@@ -89,6 +89,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Atomx](../../ads/atomx.md)
 - [Chargeads](../../ads/chargeads.md)
 - [Colombia](../../ads/colombia.md)
+- [Content.ad](../ads/contentad.md)
```

Comment:
My fault didn't catch it in the first place. Fixed in #4992 . Please check back tmr :-)